### DISCUSSION TITLE
Interrupt power reliably, how-to?

### DISCUSSION THREAD

**Discussion by aschmidt74** [arendst/Tasmota#11911] [Q&A] [answered]
As a brute means to reset some appliances I installed a sonoff-POW running tasmota.

I found a configuration that keeps the power output always ON, and a toggle (via web UI) switches the power OFF for 1s, then ON again.

However, a powerloss on the main line starts the sonoff-POW in Relay=OFF mode.

What part of a configuration am I missing to ensure a restart in Relay=ON mode?

**Comment by joba-1** [comment, marked as answer]
https://tasmota.github.io/docs/PowerOnState/

**Reply by aschmidt74** [reply]
PowerOnState=5
as I use PulseTime to time the 1s outage.
PowerOnState=1 solves the startup issue, but fails when toggeling the device when hot.

**Reply by rapi3** [reply]
With PowerOnState=5 and Pulsetime use a rule to Power ON at boot:
ON Power1#Boot DO Power1 1  ENDON

**Reply by barbudor** [reply]
I have a few devices like my ISP router with `PowerOnState 5` and `PulseTime 50` and it works as expected :
- Always power on after a power surge
- Cut the power for 5 seconds if I send a POWER OFF command and restore after the 5 seconds

PowerOnState 5 is equivalent to PowerOnState 1 but allows PuseTime to specifiy a time OFF instead of a time ON.
No need for a boot rule
It should work

**Reply by aschmidt74** [reply]
…well, that seems like my configuration, yet mine works not as expected. I'll give the rule a try.

**Comment by aschmidt74** [comment]
Sorry to say, but the rule is stored in a Rule1 set, but doesn't make sure the Relay is turned ON after a powerloss. It appears and stays OFF and needs to be toggled manually.

Any suggestions what I'm missing here?!

**Reply by rapi3** [reply]
something it is strange in your configuration.
How many relays do you have and how are defined 1, or 2...?
Is the rule 1 active ?

In my test PowerOnState 5 appear to work for relay 1 only.

try to reset your device configuration and reconfigure.
